pkgs/top-level/all-packages.nix

```diff
@@ -23108,7 +23108,18 @@ in
 
   espeakedit = callPackage ../applications/audio/espeak/edit.nix { };
 
-  eteroj.lv2 = libsForQt5.callPackage ../applications/audio/eteroj.lv2 { };
```

Comment:
this is not great for people upgrading from nixpkgs versions. ideally we'd add an alias documenting that eteroj was moved, but since it's only shortened its own attrpath that would be difficult. :confused: one more argument for putting this into a package set.

the folder referenced here should also be deleted.

Reply:
not sure that'd be a good idea. of all of these tools we only use patchmatrix and have very little (if any) experience with music in general.